### ISSUE TITLE
Basic Script not running: Use failing 500 error

### ISSUE THREAD

**Issue by iamattamai** [jonnnnyw/php-phantomjs#156]
I'm not consulting you without having spent a day on this trying every combination I could think of.
I've installed and reinstalled using the jakoch loader without error.
All the directories seem in place in the vendor and bin directories.
The client file exists.
I can run PhantomJS scripts from the command line in PUTTY successfully.
PHP is up and running on my AWS instance, other programs in PHP run fine.
However, when I run your basic example below I get a 500 error consistently. There is no other error output.
It would appear that I have a configuration issue that is preventing the Client instance from being created.
Your help is greatly appreciated.

**Comment by iamattamai** [comment]
The "basic example" referenced above is as follows:
 <?php

    use JonnyW\PhantomJs\Client;

    $client = Client::getInstance();

    /** 
     * @see JonnyW\PhantomJs\Http\Request
     **/
    $request = $client->getMessageFactory()->createRequest('http://jonnyw.me', 'GET');

    /** 
     * @see JonnyW\PhantomJs\Http\Response 
     **/
    $response = $client->getMessageFactory()->createResponse();

    // Send the request
    $client->send($request, $response);

    if($response->getStatus() === 200) {

        // Dump the requested page content
        echo $response->getContent();
    }

**Comment by Teaxtor** [comment]
How did you create your client?
Instantiating it with `$client = Client::getInstance();` is working for me.
Otherwise information about the error would be helpful. Have a look into the php log file or debug your code to identify the exact error message.

Maybe you can get additional info by having a look at the log by `$client->getEngine()->getLog()`.

**Comment by nemanjam** [comment]
Yes, I can confirm the basic example is not working, php process is stuck forever.

`public static function render_pic(){
	$client = Client::getInstance();
	$client->getEngine()->setPath(base_path().'/bin/phantomjs.exe');
	$request  = $client->getMessageFactory()->createCaptureRequest('http://jonnyw.me');
	$response = $client->getMessageFactory()->createResponse();
	$file = storage_path('app/pic.png');
	$request->setOutputFile($file);
	$client->send($request, $response);
}`

**Comment by iamattamai** [comment]
In the end I hired someone on Freelance to configure it. Too many moving parts for my knowledge level in the end and too much time invested. Thank you all for your attempts to help me. Beautiful once it's working.

**Comment by nemanjam** [comment]
Post the solution.

**Comment by iamattamai** [comment]
I don't know what the solution was in the end inasmuch as I didn't solve it myself. I imaged my AWS/EC2 instance and will just clone it for future projects to avoid the pain. Idea: There might actually be a market for pre-configured AWS templates with everything already in place, but that wouldn't be from me.

**Comment by nemanjam** [comment]
I found the error. `$request->setOutputFile($file); ' passes unescaped path. 

When I debug js script from temp folder there is `SyntaxError: \u can only be followed by a Unicode character sequence for `C:\Users\username\Desktop\app/pic.png` path.

Can author fix this?
